packages/forms-web-app/src/controllers/appeals/continue.js

```diff
@@ -0,0 +1,28 @@
+const { apiClient } = require('#lib/appeals-api-client');
+
+const appealSubmissionContinueUrls = {
+	1001: '/appeal-householder-decision/task-list',
+	1005: '/full-appeal/submit-appeal/task-list'
```

Comment:
We should reference these from constants rather than just numbers:

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/main/packages/business-rules/src/constants.js#L1

Reply:
Ah nice, didn't know about these, thanks

Reply:
ironically ended up doing the reverse and using the numbers in a type but in an ideal world he constants would be enums 🙃

Reply:
I've used the constants in this file and changed the openApi spec to have a matching enum for these, see packages/appeals-service-api/src/spec/appeal-submission.yaml